src/modules/ExchangeRates/action.js

```diff
@@ -1,7 +1,15 @@
 // @flow
 
+import { buildExchangeRates } from '../Core/selectors.js'
+import type { Dispatch, GetState } from '../ReduxTypes.js'
 const PREFIX = 'ExchangeRates/'
 export const UPDATE_EXCHANGE_RATES = PREFIX + 'UPDATE_EXCHANGE_RATES'
-export const updateExchangeRates = () => ({
-  type: UPDATE_EXCHANGE_RATES
-})
+
+export const updateExchangeRates = () => async (dispatch: Dispatch, getState: GetState) => {
+  const state = getState()
```

Comment:
Once we "build" the exchange rates asynchronously then we will send to the redux store